src/apps/Script/Script.js

```diff
@@ -797,14 +807,36 @@ const Script = React.createClass({
     );
   },
 
+  renderRawResultInNewTab() {
+    const { currentScript, lastTraceResult, lastTraceStatus } = this.state;
+    const rawResultPage = window.open();
+    const pageContent = `
+      <title>
+        ${currentScript.label} || ${lastTraceStatus}
+      </title>
+      <pre>${lastTraceResult}</pre>
+    `;
+
+    rawResultPage.document.write(pageContent);
+  },
+
   renderSidebarResultSection() {
     const styles = this.getStyles();
     const { lastTraceResult } = this.state;
+    const rightContent = (
```

Comment:
so instead, how about disabling / hiding this button when there is a `null` trace?